src/test/java/uk/gov/ons/ctp/integration/rhsvc/cloud/GCSServiceIT_Test.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.ons.ctp.integration.rhsvc.cloud;
```

Comment:
We need to maintain code coverage, and also test all routes through code, so please add some tests that assert null handling as mentioned also